assessments/guide/security-reviewer.md

```diff
@@ -1,34 +1,27 @@
 # Security Reviewer Role
 
-Reviewers will try to understand the system and probe its security.
-Specifically design level issues are meant to be addressed as well as high
-level problems with the project’s setup and operation. This is meant to 
-provide an independent analysis and estimation of the above information. 
-The goal is to ask questions to understand if there are hidden assumptions,
-underestimated risk, or design issues that harm the security of the project. 
-It may be useful to reach out to community members to understand the answers
-to some questions, especially involving deployment scenarios and the impact
- of attacks.
+Reviewers will try to understand the system and probe its security.  Specifically design level issues are meant to be addressed as well as high level problems with the project’s setup and operation. This is meant to provide an independent analysis and estimation of the above information.  The goal is to ask questions to understand if there are hidden assumptions, underestimated risk, or design issues that harm the security of the project.  It may be useful to reach out to community members to understand the answers to some questions, especially involving deployment scenarios and the impact of attacks.
 
 ## Qualifications
 
-Unless approved by the SIG chairs, at least one of the reviewers will
-have previously performed a SAFE WG audit.  (Exemptions are expected to be
-granted bootstrap the process but only in extreme cases later on.)  In
-general, a reviewer should have performed security audits for diverse
-organizations.  The ideal reviewer should also have been the recipient
-of security audits for a software project they manage.  Note that it is
-encouraged to have participation (shadowing) from participants that are not
-yet qualified to help them gain the necessary skills to be a reviewer
-in the future.
+### Required 
+
+Unless approved by the SIG chairs, at least one of the reviewers will have previously performed a CNCF project security assessment.  Exemptions to this are reviewed case by case upon established need by the CNCF Security SIG Chairs in order to bootstrap the process as appropriate.
```

Comment:
nit: thought we were going with "CNCF security assessment"

Reply:
done